.modernizrrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "minify": true,
+  "options": [
+    "setClasses"
+  ],
+  "feature-detects": [
+    "test/webgl",
+    "test/css/flexbox"
```

Comment:
We will probably want flexbox, since support is pretty good and it is such an improvement over other layout styles, but if we aren't using it presently I'm fine with waiting.